pkg/goDB/protocols/protocols_generator.go

```diff
@@ -66,15 +75,30 @@ func readProtocols() (protoList []string, err error) {
 			continue
 		}
 
-		protoList[protoID] = fields[2]
+		// discard any id that's greater than 255. It would be invalid during the
```

Comment:
Why is that necessary? AFAIK the OS Proto ID are guaranteed to be 8-bit already... :thinking:

Reply:
Urgs, I just saw https://github.com/els0r/goProbe/pull/234 - That sucks. What's proto 258 on Darwin anyway? I always thought this is a mere cosmetic thing. Maybe we should rethink the whole handling here after all, we're currently not planning for Darwin, so maybe we should wing that whole discussion / handling until we do (after all, right now we might introduce some weird cross-system incompatibilities and not even be aware of it)?